app/javascript/components/party/Candidates.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { Layout, Row, Col, Typography, Select, Avatar } from "antd";
+
+const { Title, Paragraph } = Typography;
+const { Option } = Select;
+
+class Candidates extends React.Component {
```

Comment:
You are not using this component. But we already have the information of leading candidates per party/district.

Reply:
Do we? If so, then I think we should tackle it on a separate task.

Reply:
Why? Isn't that already done in Moleiro's work?

Reply:
Is the api already available?
It was just to avoid having another mocked section. When the api is complete we could then re-enable it.

Reply:
👍